src/main/java/com/cupid/jikting/member/oauth2/userinfo/KakaoOAuth2UserInfo.java

```diff
@@ -0,0 +1,15 @@
+package com.cupid.jikting.member.oauth2.userinfo;
+
+import java.util.Map;
+
+public class KakaoOAuth2UserInfo extends OAuth2UserInfo {
+
+    public KakaoOAuth2UserInfo(Map<String, Object> attributes) {
+        super(attributes);
+    }
```

Comment:
p1. 아래처럼 어노테이션을 사용해서 코드를 좀 더 깔끔하게 할 수 있을 것 같아요!

```suggestion

@SuperBuilder
@AllArgsConstructor
public class KakaoOAuth2UserInfo extends OAuth2UserInfo {
```

+OAuth2UserInfo.class
```java
@SuperBuilder
@AllArgsConstructor
public abstract class OAuth2UserInfo {
```

Reply:
`OAuth2UserInfo`에 default 생성자가 없어서 불가능 합니다.

Reply:
아하 그렇군요!